lib/core/src/Cardano/Wallet/Primitive/Slotting.hs

```diff
@@ -12,11 +16,20 @@
 
 module Cardano.Wallet.Primitive.Slotting
     ( -- * New api using ouroboros-concensus
+      -- ** Queries
       epochOf
+    , startTime
+    , toSlotId
+    , slotRangeFromTimeRange
+    , firstSlotInEpoch
+    , ongoingSlotAt
+
+    -- ** Running queries
+    , TimeInterpreter
     , singleEraInterpreter
-    , runQuery
+    , Qry
 
-      -- * Old functions
+      -- * Legacy api
```

Comment:
Will these all be removed in a future PR, or moved to a tests-only module?

Reply:
I suppose we should remove them completely, and re-write the tests to use the new ones.

Now we are testing the old ones, and that the new ones are equivalent to the old ones (Generators aren't perfect though).